src/Expr.js

```diff
@@ -271,9 +275,13 @@ var exprToString = function(expr, caller) {
   var fn = keys[0]
   fn = convertToCamelCase(fn)
 
-  var args = keys.map(function(k) {
+  var args = []
+
+  keys.forEach(function(k) {
```

Comment:
I would probably, do it as:
`
keys
  .filter(function(k) { return expr[k] !== null })
  .map(function(k) { return exprToString(v, fn) })
`
but it can be a matter of preference anyway.

Reply:
Up to you, your approach is indeed cleaner and readability is important :)